src/App.tsx

```diff
@@ -1,60 +1,95 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [list, setList] = useState(todos);
+  const [input, setInput] = useState('');
+  const [userId, setUserId] = useState('');
+
+  const addTodo = (event: any) => {
```

Comment:
don't use any as type

Reply:
could you advise which type of event I should write?